### ISSUE TITLE
(accessibility) - color contrast violations in the left menu

### ISSUE THREAD

**Issue by Wildebrew** [angular/material.angular.io#692]
If you run [Axe](http://www.deque.com/axe) (or the Google Lighthouse audit) you will notice a large amount (around 50) of accessibility violations related to color contrast.
The problem is the gray text of the links in the left menu.

The gray text has a contrast ratio of 3.88:1 with the background but it needs to be 4.5.1, [see the WebAIM color contrast checker](https://webaim.org/resources/contrastchecker/?fcolor=797979&bcolor=F2F2F2)

(The blue text is dark enough)

## To solve

Find a color combination that passes the contrast requirements and push a global update to fix.

**Comment by Splaktar** [comment]
Thank you for your issue report. This was previously reported in https://github.com/angular/material.angular.io/issues/671 and is being worked.
